.github/workflows/release.yml

```diff
@@ -1,67 +1,30 @@
 name: Release
+
 on: [workflow_dispatch]
 
 jobs:
-  bumpversion:
-    runs-on: ubuntu-latest
-    outputs:
-      version: ${{ steps.tag_version.outputs.new_version }}
-      previous_tag: ${{ steps.tag_version.outputs.previous_tag }}
-      bump_commit_sha: ${{ steps.bumpversion.outputs.commit_hash }}
-    steps:
-      - uses: actions/checkout@v3
-      - name: Get next version
-        id: tag_version
-        uses: mathieudutour/github-tag-action@v6.0
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          default_bump: false
-          default_prerelease_bump: false
-          dry_run: true
-      - name: Set up Python 3.8
-        uses: actions/setup-python@v2
-        with:
-          python-version: "3.8"
-      - name: Create bumpversion
-        if: steps.tag_version.outputs.new_version
-        run: |
-          pip install bump2version
-          bump2version --new-version ${{ steps.tag_version.outputs.new_version }} setup.cfg
-      - name: Update Changelog
-        if: steps.tag_version.outputs.new_version
-        uses: stefanzweifel/changelog-updater-action@v1.6.2
-        with:
-          latest-version: ${{ steps.tag_version.outputs.new_tag }}
-          release-notes: ${{ steps.tag_version.outputs.changelog }}
-      - name: Commit bumpversion
-        id: bumpversion
-        if: steps.tag_version.outputs.new_version
-        uses: stefanzweifel/git-auto-commit-action@v4.14.1
-        with:
-          branch: ${{ github.ref }}
-          commit_message: "chore(release): preparing ${{ steps.tag_version.outputs.new_version }}"
-          file_pattern: CHANGELOG.md setup.cfg drydock_backups/*
   release:
-    needs: bumpversion
-    if: needs.bumpversion.outputs.version
     runs-on: ubuntu-latest
-    outputs:
-      tag: ${{ steps.tag_version.outputs.new_tag }}
-      changelog: ${{ steps.tag_version.outputs.changelog }}
+    concurrency: release
+    permissions:
+      id-token: write
+      contents: write
+
     steps:
-      - uses: actions/checkout@v3
-      - name: Create tag
-        id: tag_version
-        uses: mathieudutour/github-tag-action@v6.0
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          commit_sha: ${{ needs.bumpversion.outputs.bump_commit_sha }}
-          default_bump: false
-          default_prerelease_bump: false
-      - name: Create a GitHub release
-        if: steps.tag_version.outputs.new_tag
-        uses: ncipollo/release-action@v1
-        with:
-          tag: ${{ steps.tag_version.outputs.new_tag }}
-          name: Release ${{ steps.tag_version.outputs.new_tag }}
-          body: ${{ steps.tag_version.outputs.changelog }}
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Python Semantic Release
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
```

Comment:
I think we can start bumping this.

Lets use the latest versions of each action `checkout` and `setup-python` and use a newer python version, either 3.12 or 3.11